README.md

```diff
@@ -4,6 +4,7 @@
 [![All packages CI](https://github.com/tinkoff/taiga-ui/actions/workflows/build.yml/badge.svg?branch=main)](https://github.com/tinkoff/taiga-ui/actions/workflows/build.yml)
 [![Discord](https://img.shields.io/discord/748677963142135818?color=7289DA&label=%23taiga-ui&logo=discord&logoColor=white)](https://discord.gg/Us8d8JVaTg)
 [![angular-open-source-starter](https://img.shields.io/badge/made%20with-angular--open--source--starter-d81676?logo=angular)](https://github.com/tinkoff/angular-open-source-starter)
+[![codecov](https://codecov.io/gh/Tinkoff/taiga-ui/branch/main/graph/badge.svg?token=4A9ZQ0T7S5)](https://codecov.io/gh/Tinkoff/taiga-ui)
```

Comment:
what is the final number there? are we sure that we want to share it in readme now?

Reply:
I believe it's 60 something. Yeah, let's not show it for now and once we have a correct measure we can work on improving the number

Reply:
Okey) final number - 66.99%

![image](https://user-images.githubusercontent.com/12021443/150487741-4c3307ae-3b20-49e3-a87c-f879ac53ed7b.png)

Reply:
fixed